src/plugins/shared_ux/public/components/page_template/no_data_page/no_data_card/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiCardProps } from '@elastic/eui';
+import { MouseEventHandler, ReactNode } from 'react';
+
+export type NoDataPageActions = Partial<EuiCardProps> & {
+  /**
+   * Applies the `Recommended` beta badge and makes the button `fill`
+   */
+  recommended?: boolean;
+  /**
+   * Provide just a string for the button's label, or a whole component
+   */
+  button?: string | ReactNode;
+  /**
+   * Remapping `onClick` to any element
+   */
+  onClick?: MouseEventHandler<HTMLElement>;
+  /**
+   * Category to auto-select within Fleet
+   */
+  category?: string;
+
+  /**
+   * Description for the card. If not provided, the default will be used.
+   */
+  description?: string;
+
+  /**
+   * Layout direction of the card
+   */
+  layout?: 'vertical';
```

Comment:
I'm not sure you want to include this prop.  You've typed it as optional, but only one possible value.

Reply:
This is to make typescript play nicely.

Reply:
Added a comment in the code

Reply:
Yes, if EUI could do that, that would be great. Would you like me to open an issue?

Reply:
Sorry, no not from EUI directly. We still need to support that in EuiCard, but on line 12, you can omit this prop directly:

`Partial<Omit<EuiCardProps, 'layout'>>`